src/templates/prototypes/article.pug

```diff
@@ -24,3 +22,38 @@ block content
               li If you are asking a substantive question and have an interest to declare, tick the box and email tableoffice@parliament.uk to say what the interest is.
           li Take your question form to the Table Office or Procedural Hub in person, or ask another MP to take it for you. Or sign the question form and post it or ask your staff to take it to the Table Office or Procedural Hub (faxed, stamped or photocopied signatures cannot be accepted).
           li The Table Office will check your question and contact you if they have a problem. Once the Table Office has approved your question, it has been tabled.
+
+      aside
+        .block.theme--grey-4
+          h2 Related Articles
+          ul.list--pipe
+            li
+              .list--details
+                +link('#', 'MPs guide to business questions')
+            li
+              .list--details
+                +link('#', 'MPs guide to deadlines for questions')
+            li
+              .list--details
+                +link('#', 'An overview of oral questions for MPs')
+            li
+              .list--details
+                +link('#', 'MPs guide to Prime Ministers questions')
+            li
+              .list--details
+                +link('#', 'MPs guide to problems with questions')
+            li
+              .list--details
+                +link('#', 'MPs guide to rules for questions')
+            li
+              .list--details
+                +link('#', 'MPs guide to urgent questions')
+            li
+              .list--details
+                +link('#', 'MPs guide to written questions')
+
+        .block.theme--grey-4
+          h2 Save and share
+          figure.svg--inline(data-file-print='print')
```

Comment:
Better to use a definition list here. You won't need `.svg--inline` then

Reply:
I've removed `vertical-align: middle;` also from the _btn_ mixin's svg element.